airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -104,7 +103,7 @@ export const connectionValidationSchema = yup
                 name: "cronExpression",
                 message: "Incorrect cron string",
                 test(value) {
-                  if (value && isValidCron(value, { seconds: true, allowBlankDay: true })) {
+                  if (value) {
```

Comment:
Is this an emergency patch? Otherwise I think we should fix the isValidCron method.

Reply:
So I believe this was added from a suggestion I made on the cron PR that it might be nice to validate cron strings in the webapp so that users got faster feedback. But I think Harshith used code from a third-party plugin to perform this validation, which may have some differing cron-string validation logic.

Since our backend server is performing cron string validation on form submission using the same library that we use to actually process those cron strings for scheduling purposes, I think it is reasonable to just let the backend perform these validations and disable the validation being done in the webapp for now. 

We may want to just remove the webapp validation code entirely, as it will probably be too cumbersome to maintain two separate implementations of cron validation logic -- we should just have a singular place that performs these validations, in the backend.

Reply:
We should remove the util then so we don't have bad and unused code lying around.

Reply:
I've put up a PR for that here: https://github.com/airbytehq/airbyte/pull/16792